### DISCUSSION TITLE
"_ssgManifest.js" & _buildManifest.js missing for a next app hosted on Gcloud

### DISCUSSION THREAD

**Discussion by ashish-shrma** [vercel/next.js#27442] [Help]
sometimes the console shows these errors on opening the website

<img width="1421" alt="Screenshot 2021-07-23 at 8 45 09 PM" src="https://user-images.githubusercontent.com/9575370/126803817-d7a6b443-e87b-40c5-9484-30cc88c2d0b8.png">

the app does use ISR and that seems to be working, it does get updated, what do these files do? what could happen if they are missing? 

- "react": "17.0.2"
- "next": "10.1.3",
- node "15.1.2"

**Comment by leerob** [comment]
Can you provide a reproducible example?
